dlt/helpers/dashboard/utils.py

```diff
@@ -39,6 +45,20 @@
 #
 
 
+def _exception_to_string(exception: Exception) -> str:
+    """Convert an exception to a string"""
```

Comment:
which other exceptions can be encountered that we can convert to a better version on the dashboard UI when connecting to a destination or instantiating the destination client?